docs/developer_guide/getting_started.md

```diff
@@ -14,20 +14,19 @@ Make sure you have Node.js and npm installed; you need them to obtain and use th
 
 ### Create a new Node.js project
 1. Create and enter a new directory to hold the project:
-  ### Linux/Mac
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-
+  1. `$ mkdir ~/offline_wallet_generator`
+  2. `$ cd ~/offline_wallet_generator`
+<!--
   ### Windows
   1. `mkdir C:\offline_wallet_generator`
   2. `cd C:\offline_wallet_generator`
-
+-->
```

Comment:
Is that even valid markup?
Anyway. Why not delete that section?

Reply:
It's valid html... and it keeps those lines from displaying in the file.
Is there markup notation for comments?
In any case, *siiiigh* yes, I should and will just remove it. I was hesitant because i put a lot of work into converting the shell scripts into batch files and catering to windows users in the instructions. It was only after I did all of this that @woodser convinced me that I was mostly wasting my time, because Windows users apparently make up an insignificant portion of the target audience.